lib/routes/users-login.js

```diff
@@ -33,7 +33,8 @@ module.exports = (server, options) => {
 
             const foundUser = await Users.query() .where({ email: Payload.email }).first();
```

Comment:
What I was suggesting was changing this line to `await Users.query().where({ email: Payload.email }).whereNotNull(password).first();` so that if password was null, we wouldn't return a user at all.

Reply:
Again, either way is fine, they both accomplish the same thing, just personal preference for offloading work onto the DB.

Reply:
Oh yeah nice!